rs/execution_environment/src/canister_manager.rs

```diff
@@ -2513,7 +2643,7 @@ impl From<CanisterManagerError> for UserError {
                 canister_id,
                 controllers_expected,
                 controller_provided } => {
-                let controllers_expected = controllers_expected.iter().map(|id| format!("{}{additional_help}", id)).collect::<Vec<String>>().join(" ");
+                let controllers_expected = controllers_expected.iter().map(|id| format!("{}", id)).collect::<Vec<String>>().join(" ");
```

Comment:
This was a mistake from a previous PR that went unnoticed because `additional_help` was the empty string.

Reply:
An existing test caught this once we started having a real value for `additional_help` which is how I noticed it and fixed it here. I think it was this one: https://github.com/dfinity/ic/pull/1218/files/8ae41887318953cb6731bc0cab939258ab559b0a#diff-ad28b50d7fe9d7c4ae973952ee2e39944eb79420e2b6f0f76e16fcb481871656R338-R345